.github/workflows/stale-tasks.yml

```diff
@@ -0,0 +1,26 @@
+# This is a workflow that runs periodically to mark issues/prs with no recent activity as stale, it also closes/removes label based on following activity
+
+name: Mark and close stale issues/PRs
+on:
+  schedule:
+  - cron: '0 0 * * *' # runs daily at 12:00 am UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-label: stale
+          stale-pr-label: stale
+          days-before-stale: 45
+          days-before-close: 45
+          remove-stale-when-updated: true
+          stale-issue-message: > 
+            Issue has been automatically marked as stale because it has been open for 45 days with no activity. Issue will be closed if no further activity occurs.
+            Thank you for your contributions.
```

Comment:
Is this better or "Issue has been automatically marked as stale due to inactivity for 45 days. Update the issue to remove label, otherwise it will be automatically closed."

Reply:
i like what you have in the PR better, but either is good.